plenum/server/pool_manager.py

```diff
@@ -238,20 +238,21 @@ def _updateNode(txn_data):
                 if BLS_KEY in txn_data[DATA]:
                     self.node_blskey_changed(txn_data)
 
-        seqNos, info = self.getNodeInfoFromLedger(nodeNym)
-
         # `onPoolMembershipChange` method can be called only after txn added to ledger
-        if len(seqNos) == 0:
+        if self.ledger.getBySeqNo(get_seq_no(txn)) is None:
+            self._set_node_order(nodeNym, node_services=txn_data[DATA].get(SERVICES, None))
```

Comment:
We call _set_node_order  to bring state _ordered_node_services in the same way as _ordered_node_ids. To add to both dicts data of a new transaction. But this is not important and can be removed.
We can remove this check but if error will happen it will not be checked. If it's ok, I will remove this check.

Reply:
Removed.